next.config.js

```diff
@@ -2,7 +2,12 @@ const path = require('path');
 
 module.exports = {
   webpack: config => {
-    config.resolve.alias['rifm'] = path.resolve('./dist/rifm.esm.js');
+    const rifmPath =
+      process.env.NODE_ENV === 'production'
+        ? './dist/rifm.esm.js'
+        : './src/index.js';
```

Comment:
Does source work without next babel config extension?

Reply:
Yep it works, even reloads on change

Reply:
Then let's consume only src. I used dist to consume released version.

Reply:
done